pkg/security/admission/admission.go

```diff
@@ -183,6 +183,28 @@ func assignSecurityContext(provider scc.SecurityContextConstraintsProvider, pod
 
 	// Note: this is not changing the original container, we will set container SCs later so long
 	// as all containers validated under the same SCC.
+	for i, containerCopy := range pod.Spec.InitContainers {
```

Comment:
This isn't the factorization I expected.  Shouldn't `InitContainers` and `Containers` be handled the same way with a different path?